### ISSUE TITLE
Closed runway area changes with LVP status

### ISSUE THREAD

**Issue by kusterjs** [LB767/vSMR#13]
I observed that the closed runway area is changed with the LVP status. However, the closed runway area should always be displayed in the same dimensions. Only the area considered for RIMCAS warning should change, but this is an area not to be displayed at all. Runway data could be taken from the sct file directly. Then, only the width for normal visibility and LVP would need to be specified additionally in the vSMR settings file.

**Comment by LB767** [comment]
Are you sure you would prefer the RIMCAS areas (normal and LVP) to be defined by width only instead of coordinates? It makes for less flexibility when defining the areas, but if that's fine for you then I'll make the change.

Edit: for reference, this is what the runway definitions from the sector file look like in Zürich, unless they are wrong. 
![image](https://user-images.githubusercontent.com/20526358/78569499-036f4100-7824-11ea-846c-e9de2bcf4c85.png)

**Comment by kusterjs** [comment]
Right, flexibility is definitely some that would be lost. I guess then the runway can be a polygon of also more than 4 coordinates? This is definitely an advantage, especially for the definition of the LVP area. But there should be 3 kinds of polygons:
1. area to show a closed runway
2. normal RIMCAS protection area
3. optional: LVP RIMCAS protection area

A few notes here:
- The structure in the "maps" part is not according the original wiki, where the tag "runways" is not mentioned. Is it still required and an error in the wiki? Or can this section be defined without using the "runways" section?
- Where is the "runway_name" used? I can not find this name in the plugin HMI. Is it just an internal name?
- An additional LVP RIMCAS protection area should be optional. If not defined, the normal RIMCAS are for the runway in question shall be used also when LVP is set on.
- Only the "closed runway" polygon should be visible for the controller. It's color seems currently not configurable, is this intentionally?
- The drawing of the closed runway is currently on top of the message box of EuroScope, while the remaining drawing of the ground maps is covered by this box.
![image](https://user-images.githubusercontent.com/10661679/78603125-ddfc2a80-7857-11ea-8c9d-c23ba508f48c.png)

**Comment by LB767** [comment]
> I guess then the runway can be a polygon of also more than 4 coordinates?

I'm not sure if this is possible in the current version, but if needed I could most likely implement it without too much work,

To answer your points:

- Not sure I understood what you meant here, the wiki does show the `runways` section 
![image](https://user-images.githubusercontent.com/20526358/78607703-e0628280-785f-11ea-8406-61228d705a4f.png)
and the profile.json also has it (unless yours is different?)

- The `runway_name` is used to know which runway the coordinates specified in the profile represent. vSMR checks if custom coordinates are given for all runways taken from the sector file, and if so uses these custom coordinates for RIMCAS instead of the sector file ones. The name is also what is displayed in the IAW.

- Noted, I'll add this to the code.

- The color is hard-coded yes, I could make it configurable.

- Nice bug... I'll try and fix it.
